src/pysdmx/model/message.py

```diff
@@ -0,0 +1,35 @@
+"""Message module.
+
+This module contains the enumeration for the different types of messages that
+can be written.
+"""
+
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+from msgspec import Struct
+
+
+class ActionType(Enum):
+    """ActionType enumeration.
+
+    Enumeration that withholds the Action type for writing purposes.
+    """
+
+    Append = "append"
+    Replace = "replace"
+    Delete = "delete"
+    Information = "information"
+
+
+class Header(Struct, frozen=True, kw_only=True):
+    """Header for the SDMX messages."""
+
+    id: str = "test"
+    test: bool = True
+    prepared: datetime = datetime.strptime("2021-01-01", "%Y-%m-%d")
```

Comment:
Sorry for not spotting this earlier but, I think there are 2 issues with this:
- It should be a time stamp (i.e. date and time), not a date.
- Maybe the date should not be hardcoded, but should instead be the time when the Header gets instantiated, in the UTC timezone?

Reply:
My best bet would be to use datetime.now() as default value, but is a nightmare to get the 100% code coverage. I will have another look at it

Reply:
Changed to datetime.now() on latest commit.